src/lib/core/styles/tokens/divider/_tokens.scss

```diff
@@ -0,0 +1,15 @@
+@use 'sass:map';
+@use '../../theme';
+@use '../../utils';
+@use '../../border';
+
+$tokens: (
+  color: utils.module-val(divider, color, theme.variable(outline)),
+  width: utils.module-val(divider, width, border.variable(thin)),
+  border-style: utils.module-val(divider, border-style, solid),
+  margin: utils.module-val(divider, margin, 0),
```

Comment:
The more I think about this, I think we may want to just remove this `margin` token and the `margin` style from within the element.. See my other comment about the `:host-context` as well.